src/css/colors.scss

```diff
@@ -1,14 +1,15 @@
+$color-primary: #5bc0be;
```

Comment:
Why?

Reply:
moved arround. Colors that are used more often should appear at the top